src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+const Big = require('big.js');
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result = new Big(0);
+  const n1 = new Big(numberOne);
+  const n2 = new Big(numberTwo);
+  switch (operation) {
+    case '÷':
+      result = n1.div(n2);
+      break;
```

Comment:
Dont forget to manage division by ZERO on next milestone with this, because it will through an error 😊

Reply:
:+1:  OMG. Thank you @shloch for this pointer